bulk_site_creator/views.py

```diff
@@ -178,6 +181,7 @@ def create_bulk_job(request: HttpRequest) -> Optional[JsonResponse]:
     school_id = sis_account_id.split(":")[1]
 
     table_data = json.loads(request.POST['data'])
+    print(f'Table Data: {table_data}')
```

Comment:
A relic from testing? Or if not, maybe use the logger?

Reply:
Yea, good catch